src/databricks/labs/ucx/aws/access.py

```diff
@@ -49,33 +50,35 @@ def __init__(
         self._kms_key = kms_key
         self._filename = self.INSTANCE_PROFILES_FILE_NAMES
 
-    def create_uc_roles_cli(self, *, single_role=True, role_name="UC_ROLE", policy_name="UC_POLICY"):
+    def uc_roles_list(self, *, single_role=True, role_name="UC_ROLE", policy_name="UC_POLICY"):
```

Comment:
nit: should be a verb
```suggestion
    def list_uc_roles(self, *, single_role=True, role_name="UC_ROLE", policy_name="UC_POLICY"):
```